modules/common/profiles/release.nix

```diff
@@ -8,7 +8,9 @@ let
 in
 {
   options.ghaf.profiles.release = {
-    enable = mkEnableOption "release profile";
+    enable = (mkEnableOption "release profile") // {
+      default = true;
```

Comment:
our default is false. the release builds have some significant issues at the moment so all our testing is on the debug targets

Reply:
fixed.